### PR TITLE
C365623 Verify that "Provider name" is renamed (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by artsiomBandarenka**
[C365623](https://foliotest.testrail.io/index.php?/cases/view/365623)
[report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4576/allure/)

### CODE REVIEW

**Review by dnovikau — APPROVED** [review]

File: cypress/support/fragments/settings/remote-storage/configurations.js [hunk 1]
```diff
@@ -65,12 +65,15 @@ function openCreateConfigurationForm() {
 
 const configurations = {
   DematicEMS: {
-    title: 'Dematic EMS',
+    title: REMOTE_STORAGE_PROVIDER_NAMES.Dematic_EMS,
```
Comment (nayimovag) [inline cypress/support/fragments/settings/remote-storage/configurations.js:68]: lets keep this consistent for other configuration titles too

File: cypress/support/fragments/settings/remote-storage/configurations.js [hunk 2]
```diff
@@ -308,4 +311,18 @@ export default {
   confirmCreateRemoteStorage() {
     cy.do(confirmationModal.find(saveBtn).click());
   },
+  opentEditConfigurationForm(name) {
+    cy.do([
+      MultiColumnListCell({ content: name }).click(),
+      Pane({ title: name }).find(actionsBtn).click(),
+      Button({ id: 'clickable-edit-storage' }).click(),
```
Comment (nayimovag) [inline cypress/support/fragments/settings/remote-storage/configurations.js:318]: let's move repeated locators in the file into const variables

**Review by nayimovag — CHANGES_REQUESTED** [review]

File: cypress/support/fragments/settings/remote-storage/configurations.js (hunk 1 above)
Comment (artsiomBandarenka) [reply cypress/support/fragments/settings/remote-storage/configurations.js:70]: done

File: cypress/support/fragments/settings/remote-storage/configurations.js (hunk 2 above)
Comment (artsiomBandarenka) [reply cypress/support/fragments/settings/remote-storage/configurations.js]: done

**Review by nayimovag — APPROVED** [review]

**Pull request merged**